terraform/groups/ecs-service/main.tf

```diff
@@ -53,7 +53,7 @@ module "ecs-service" {
   # Docker container details
   docker_registry   = var.docker_registry
   docker_repo       = local.docker_repo
-  container_version = var.lfp_pay_web_version
+  container_version = var.penalty_payment_web_version
```

Comment:
Again, might be worth leaving this off until we get the pipeline stuff sorted.

Reply:
Revert back to lfp_pay_web as suggested